.github/workflows/build-test-deploy.yml

```diff
@@ -0,0 +1,31 @@
+# This file is maintained in the Qontigo/Tech-solutions team and will test the code changes on the fly
+
+# Builds and tests code on different versions of Python
+
+name: Build and test the code
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test_pull_request:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ "3.6", "3.7", "3.8", "3.9", "3.10", "3.11" ]
```

Comment:
any reason why python 3.12 isnt in this list?

Reply:
Added now, missed it by mistake! Thank you for catching that!